src/Nav/Navbar.tsx

```diff
@@ -19,7 +20,15 @@ function Navbar() {
       variant="dark"
     >
       <LinkContainer to="/">
-        <ReactNavbar.Brand>PuzzTool</ReactNavbar.Brand>
+        <ReactNavbar.Brand>
+          <div className="NavbarLogo">
+            <img
+              className="NavbarLogo-img"
+              src={require('../Images/puzztool_banner_white.svg')}
```

Comment:
Import the image above and reference it here. We don’t use require, iirc and I definitely don’t recommend doing it inline.